src/routes/entry-point.ts

```diff
@@ -0,0 +1,20 @@
+import Router from '@koa/router';
+import http from 'http2';
+import { Context, Middleware, Next } from 'koa';
+import { schema } from 'rdf-namespaces';
+import url from 'url';
+import Routes from './index';
+
+export default (router: Router): Middleware => (
+  async ({ request, response }: Context, next: Next): Promise<void> => {
+    response.status = http.constants.HTTP_STATUS_OK;
+    response.body = {
+      '@id': url.resolve(request.origin, router.url(Routes.EntryPoint, {})),
+      '@type': schema.EntryPoint,
+      [schema.name]: { '@value': 'Article Store', '@language': 'en' },
```

Comment:
first time I see a object key being an array? (if that's what it is)

Reply:
Which part? If you mean the property name, it's escaped as can't declare it with a `.` inside.

Reply:
Difference with ``schema.name`:`?

Reply:
Ah it's a lookup, would require another assignment then like `let schemaName = schema.name'`